### ISSUE TITLE
[BUG] [AKS/EKS] Do not install Filebeat when there is no Elasticsearch

### ISSUE THREAD

**Issue by to-bar** [epiphany-platform/epiphany#1870]
**Describe the bug**
Filebeat is installed even when `specification.components.logging.count: 0`.

**To Reproduce**
1. Deploy Epiphany on AKS/EKS cluster (`k8s_as_cloud_service: true`) with no `logging` component.
2. Check status of filebeat pods.

**Expected behavior**
Skip Filebeat installation when `logging` group doesn't exist or is empty.

**OS (please complete the following information):**
- All

**Cloud Environment (please complete the following information):**
- AKS
- EKS

**Additional context**
Extend condition for `Include installation tasks for Filebeat as DaemonSet for "k8s as cloud service"` task.

In such configuration each Filebeat pod is restarted in a loop unnecessarily reserving resources:
```
State:          Waiting
  Reason:       CrashLoopBackOff
Last State:     Terminated
  Reason:       Error
  Exit Code:    1
  Started:      Fri, 20 Nov 2020 20:47:45 +0100
  Finished:     Fri, 20 Nov 2020 20:47:45 +0100
Ready:          False
Restart Count:  37
Limits:
  cpu:     1
  memory:  200Mi
Requests:
  cpu:     100m
  memory:  100Mi
```

**Comment by to-bar** [comment]
After solving this issue, the fix should be backported to v0.8 branch.

**Comment by przemyslavic** [comment]
Fixed. Filebeat installation as DaemonSet is skipped if there is no logging component.
